### ISSUE TITLE
Commissioning a large number of devices does not function the same in all ecosystems

### ISSUE THREAD

**Issue by sander-gitl** [sammachin/node-red-matter-bridge#11]
When commissioning a large (51) number of devices, this does not function in all ecosystems. 

In some system(s), all devices are commissioned without error.
In other system(s), an error is displayed, and while devices are seemingly commissioned : when trying to remove the bridge, an error is displayed.

Attached is a sample flow that was used for testing:
[exampleflow_rn002_sample.json](https://github.com/user-attachments/files/17716842/exampleflow_rn002_sample.json)

Expectation: ecosystems perform the same way when commissioning a large number of devices.

**Comment by Apollon77** [comment]
Can you please be more Presice? Which ecosystem s have issues?  It is "known" that some ecosystems have "limits" and work better with a lower number of devices.

So IF it works with one ecosystem then if technically also works with the others - unless they themself or your host has issues ... So said this Please provide a debug log from a case where you see issues with a specific ecosystem and tell which. Thank you

**Comment by sammachin** [comment]
Seeing some problems with my own alexa system that may be related to this, I have 19 device on the bridge and although they've all be discovered a few of them show as unavailble.
I've added most of these devices **after** commisioning the bridge.
They all show up fine in my apple controller.

**Comment by sammachin** [comment]
Did some testing of this scenario:
I created a new bridge and added 64 devices to that, all on-off lights.
Then tried commisioning the bridge for the first time into Apple, Google and Amazon ecosystems separatly (resetting everything between tests)
Apple Commisioned fine.
Google Commisioned fine.
Amazon looked to be discovering devices in the debug log but the commisioning app timed out and threw error RN002,
However once I then went into the Alexa app some of the devices were there (approx 40) and after a few minutes it had updated and discovered all 64.

Therefore I would thing that all 3 major platforms support at least 64 devices on a bridge, however Amazon has a timeout in the mobile app but devices are successfully discovered and added to the controller eventually.

For a 2nd test I then added a 65th device and all 3 platforms discovered and added this without further trouble.

I also tested amazon by commisioning a bridge with a single device and then adding another 63 devices, again this discovered all the new devices after a few minutes and was running happily.

So the reccomendation with Amazon would be to commision the bridge initially with only 1 device and then add the addional ones.

Going to close this issue as I don't believe its an issue with node-red-matter-bridge nor matter.js
